setup/tue-functions.bash

```diff
@@ -19,6 +19,66 @@ function _list_subdirs
     done
 }
 
+function _set_export_option {
+
+    # _set_export_option KEY VALUE FILE
+    # Add the following line: 'export KEY=VALUE' to FILE
+    # Or changes the VALUE to current value if line already in FILE.
+
+    key=${1//\//\\/}
+    value=${2//\//\\/}
+    sed -i \
+        -e '/^#\?\(\s*'"export ${key}"'\s*=\s*\).*/{s//\1'"${value}"'/;:a;n;ba;q}' \
+        -e '$a'"export ${key}"'='"${value}" $3
+}
+
+# ----------------------------------------------------------------------------------------------------
+#                                              SSH
+# ----------------------------------------------------------------------------------------------------
+
+function tue-use-ssh
```

Comment:
I think this should be moved to tue-env as it is more of an environment property than that of tue-functions.bash